tmscti/ai_control/tms_ai_cycle_waypoints.sqf

```diff
@@ -0,0 +1,34 @@
+_group = (_this select 0);
```

Comment:
Please rename these files so they do not have the ```tms_ai_``` prefix. Since they are inside the directory ``` tmscti/ai_control/```, this prefix is superfluous.